Client/Frontend/UserContent/UserScripts/MainFrame/AtDocumentStart/TrackingProtectionStats.js

```diff
@@ -35,20 +35,20 @@ function install() {
     }
   })
 
-  function sendMessage(url) {
+  function sendMessage(url, resourceType) {
     if (url) {
-      webkit.messageHandlers.trackingProtectionStats.postMessage({ url: url });
+      webkit.messageHandlers.trackingProtectionStats.postMessage({ url: url, resourceType: resourceType === undefined ? null : resourceType });
```

Comment:
Should we have a resource type `other` maybe?

Reply:
I considered it but in the end it doesn't make much sense because if we don't know the resource type, then we aren't going to track it. (`other` would just be a synonym for `nil` at that point)